docker-compose.yaml

```diff
@@ -476,7 +476,7 @@ services:
         volumes:
             - ./mongo-data:/data/db
     nginx:
-        image: nginx:1.17-alpine
+        image: nginx:1.23-alpine
```

Comment:
these upgrades should be part of separate PR, since this concerns only firebase

Reply:
> Please isolate the docker container upgrades from this PR and add the env vars also in compose where needed
Sorry! Pushed wrong docker-compose file!